docs/development/mono-repository.md

```diff
@@ -0,0 +1,120 @@
+
+```js
+// /workspace-a/package.json
+{
+  "name": "workspace-a",
+  "version": "1.0.0",
+  "dependencies": {
+    "foo": "0.0.1"
+  }
+}
+
+// /packages/foo/package.json
+{
+  "name": "foo",
+  "version": "1.0.0"
+}
+```
+
+#### Run `yarn install` in the workspace root
+
+
+Given the above, we should expect the following folder structure
+```
+/package.json
+/yarn.lock
+/workspace-a/package.json
+/packages/foo/package.json
+/node_modules
+/node_modules/workspace-a -> /workspace-a
+```
+
+## Guidance
+
+There are a handful of benefits and challenges to monorepos.
+
+With a monorepo, it is recommended to leverage a common set of tooling for [linting][linting] and [testing][testing] across all packages in the repository. See our [code-consistency][code-consistency] section for further guidance. Having a common set of tooling is a benefit for monorepos since each package managed in the repo does not need its set of tooling or versioning. On the other hand, teams can also control scripts individually per artifact or package if necessary. See `Advanced Tips / Tricks` section for more details.
+
+While there are a benefits, there are challenges to overcome.
+
+* Detecting what to build. Build times can be long due to overbuilding the entire repository
+* Long build times. Because of having to build the entire repository, this will cause build times to increase because tooling no longer is just building one artifact/package, but many different ones.
+* Not just JavaScript. A monorepo may not just be JavaScript, and can impact organization, tooling, and how to approach building different artifacts or packages in different languages.
+* Git repositories can grow larger in size.  Since we are building and maintaing different packages in a singular repository, we can exponentially grow the size of the git repository.
+* Commit messages can be harder to interpret due to multiple artifacts, packages, or teams.
+
+
+### Building
+
+When building npm packages (or any type of package) (the build) in the monorepo, we recommend:
+
+* [bazel][bazel]
+* Custom scripts
+
+Using appropriate build tools help overcome challenges state previously. With build tools, we are able to detect file changes and build the appropriate artifact/packages necessary without building the entire repository. This helps reduce long build times. Having a good build tool like [bazel][bazel] also allows for potential to build out other project artifact/packages in different languages.
+
+When committing messages, teams can utilize [conventionalcommits][conventionalcommits] to reduce confusion when working on a monorepo.
+
+### Publishing
+
+If you are working with a set of packages that need to be published, you can leverage:
+
+* [changesets][changesets]
+* [lerna][lerna] - Was previously used but because of [yarn workspaces][yarn-workspaces] and [npm workspaces][npm-workspaces], teams have no longer found a need for [lerna][lerna]
```

Comment:
as lerna is not recommended, it should be typographically distinct from the current recommendation
```suggestion

About [lerna][lerna]: Lerna was previously used but because of [yarn workspaces][yarn-workspaces] and [npm workspaces][npm-workspaces], teams have no longer found a need for [lerna][lerna]
```